src/main/scala/akka/persistence/cassandra/query/CassandraReadJournalConfig.scala

```diff
@@ -0,0 +1,23 @@
+package akka.persistence.cassandra.query
+
+import scala.concurrent.duration._
+import com.typesafe.config.Config
+import com.datastax.driver.core.ConsistencyLevel
+import akka.persistence.cassandra.journal.CassandraJournalConfig
+
+private[query] class CassandraReadJournalConfig(config: Config, writePluginConfig: CassandraJournalConfig) {
```

Comment:
Me neither, it is pretty minor. I just wanted to check that a client that only wants to use queries does not have to set properties that are specific for write journal they don't really care about. Otherwise I am happy with it.